projects/1.16/assets/projectex-forge/projectex/lang/zh_cn.json

```
@@ -0,0 +1,111 @@
+{
+	"block.projectex.alchemy_table": "炼金术桌",
+	"block.projectex.basic_collector": "基础能量收集器1型",
+	"block.projectex.basic_power_flower": "基础力量花盆栽罐1型",
+	"block.projectex.basic_relay": "基础反物质继电器1型",
+	"block.projectex.blue_collector": "蓝物质能量收集器8型",
+	"block.projectex.blue_power_flower": "蓝物质力量花盆栽罐8型",
+	"block.projectex.blue_relay": "反蓝物质继电器8型",
+	"block.projectex.collector.emc_produced": "EMC生产速度：%s/s",
+	"block.projectex.collector.tooltip": "对服务器的TPS很友善。每秒只产生一次EMC。",
+	"block.projectex.compressed_refined_link": "压缩精致EMC接口",
+	"block.projectex.compressed_refined_link.tooltip": "与精致EMC接口相似，但它有54个输出槽。",
+	"block.projectex.cyan_collector": "青物质能量收集器9型",
+	"block.projectex.cyan_power_flower": "青物质力量花盆栽罐9型",
+	"block.projectex.cyan_relay": "反青物质继电器9型",
+	"block.projectex.dark_collector": "暗物质能量收集器2型",
+	"block.projectex.dark_power_flower": "暗物质力量花盆栽罐2型",
+	"block.projectex.dark_relay": "反暗物质继电器2型",
+	"block.projectex.energy_link": "能量EMC接口",
+	"block.projectex.energy_link.tooltip": "你可以用这个方块通过EMC收集器来为你的转化桌增加EMC。",
+	"block.projectex.fading_collector": "渐消物质能量收集器15型",
+	"block.projectex.fading_power_flower": "渐消物质力量花盆栽罐15型",
+	"block.projectex.fading_relay": "反渐消物质继电器15型",
+	"block.projectex.final_collector": "终极能量收集器16型",
+	"block.projectex.final_power_flower": "终极力量花盆栽罐16型",
+	"block.projectex.final_relay": "终极反物质继电器16型",
+	"block.projectex.green_collector": "绿物质能量收集器10型",
+	"block.projectex.green_power_flower": "绿物质力量花盆栽罐10型",
+	"block.projectex.green_relay": "反绿物质继电器10型",
+	"block.projectex.lime_collector": "黄绿物质能量收集器11型",
+	"block.projectex.lime_power_flower": "黄绿物质力量花盆栽罐11型",
+	"block.projectex.lime_relay": "反黄绿物质继电器11型",
+	"block.projectex.magenta_collector": "品红物质能量收集器4型",
+	"block.projectex.magenta_power_flower": "品红物质力量花盆栽罐4型",
+	"block.projectex.magenta_relay": "反品红物质继电器4型",
+	"block.projectex.orange_collector": "橙物质能量收集器13型",
+	"block.projectex.orange_power_flower": "橙物质力量花盆栽罐13型",
+	"block.projectex.orange_relay": "反橙物质继电器13型",
+	"block.projectex.personal_link": "个人EMC接口",
+	"block.projectex.personal_link.tooltip": "与能量EMC接口相似，但也能允许你放入与取出物品。",
```

```suggestion
	"block.projectex.personal_link.tooltip": "与基础能量EMC接口相似，还能存入和取出物品。",
```
```suggestion
	"block.projectex.personal_link.tooltip": "比基础能量EMC接口多了存入和取出物品的功能",
```
应为 not only... but also...，的省略。“但也”？

[物品/方块] 基础能量EMC接口 (Basic Energy EMC Link) - Project EX 
https://www.mcmod.cn/item/216852.html